control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -1163,6 +1157,39 @@ func (r *reconciler) reconcileOpenshiftOAuthAPIServerAPIServices(ctx context.Con
 	return errors.NewAggregate(errs)
 }
 
+func (r *reconciler) reconcileKASEndpoints(ctx context.Context, hcp *hyperv1.HostedControlPlane) error {
```

Comment:
I really wish we had some way to test the whole reconciler together so we could capture that it now does this. @sjenning